ipn/ipnlocal/c2n.go

```diff
@@ -48,6 +48,8 @@ var c2nHandlers = map[methodAndPath]c2nHandler{
 	req("/debug/metrics"):           handleC2NDebugMetrics,
 	req("/debug/component-logging"): handleC2NDebugComponentLogging,
 	req("/debug/logheap"):           handleC2NDebugLogHeap,
```

Comment:
change this line's value too and delete the `handleC2NDebugLogHeap` variable? Oh, but then you'd be deleting the "gc" parameter which this isn't rebased to include?

In any case, /debug/logheap and /debug/pprof/heap should have identical behavior (and both support the gc parameter. or at least the pprof version should)

Reply:
I decided to go all out here and just make the `/debug/pprof/` endpoint behave exactly like the real `/debug/pprof`. This required me to introduce a ServeMux, but I left the old pattern matching in place for compatibility.